<launch>
<node pkg="solar_panel_project" type="RGB_visual_ros_sim_reordered.py" name="RGB_visual_ros_sim" output="screen"/>

<!--node pkg="solar_panel_project" type="parameters_optimization_task.py" name="parameters_optimization_task" output="screen"/-->

<node name="RGB_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/camera_vision_RGB_output" />
</node>

<group ns="RGB_params">
      <rosparam file="$(find solar_panel_project)/config/RGB_detection_params.yaml" />



 </group>
</launch>
